Makefile

```
@@ -81,34 +87,36 @@ oc-all-clean:
 	test -n "$(APP_NAME)" # Please provide an app name via APP_NAME=openshift-launchpad
 	@echo "+\n++ Tearing down all OpenShift objects created from templates...\n+"
 	@oc project $(NAMESPACE)
-	@oc delete all -l app=$(APP_NAME)
+	@oc delete all -l app=$(APP_NAME) --ignore-not-found
+	@oc delete secret $(APP_NAME)-database --ignore-not-found
 
 oc-database-clean:
 	test -n "$(NAMESPACE)" # Please provide a namespace via NAMESPACE=myproject
 	test -n "$(APP_NAME)" # Please provide an app name via APP_NAME=openshift-launchpad
 	@echo "+\n++ Tearing down OpenShift postgresql objects created from templates...\n+"
 	@oc project $(NAMESPACE)
-	@oc delete all -l template=$(APP_NAME)-database
+	@oc delete all -l template=$(APP_NAME)-database --ignore-not-found
+	@oc delete secret $(APP_NAME)-database --ignore-not-found
 
 oc-persisted-clean:
 	test -n "$(NAMESPACE)" # Please provide a namespace via NAMESPACE=myproject
 	test -n "$(APP_NAME)" # Please provide a database service name via DATABASE_SERVICE_NAME=db-service
 	@echo "+\n++ Remove persistant storage used by db service \n+"
 	@oc project $(NAMESPACE)
-	@oc delete pvc $(APP_NAME)-database
-	@oc delete secret $(APP_NAME)-database
-	@oc delete nsp -l app=$(APP_NAME)
+	@oc delete pvc $(APP_NAME)-database --ignore-not-found
+	@oc delete secret $(APP_NAME)-database --ignore-not-found
+	@oc delete nsp -l app=$(APP_NAME) --ignore-not-found
 
 oc-server-clean:
 	test -n "$(NAMESPACE)" # Please provide a namespace via NAMESPACE=myproject
 	test -n "$(APP_NAME)" # Please provide an app name via APP_NAME=openshift-launchpad
 	@echo "+\n++ Tearing down OpenShift server objects created from templates...\n+"
 	@oc project $(NAMESPACE)
-	@oc delete all -l template=$(APP_NAME)-server
+	@oc delete all -l template=$(APP_NAME)-server --ignore-not-found
 
 oc-client-clean:
 	test -n "$(NAMESPACE)" # Please provide a namespace via NAMESPACE=myproject
 	test -n "$(APP_NAME)" # Please provide an app name via APP_NAME=openshift-launchpad
 	@echo "+\n++ Tearing down OpenShift client objects created from templates...\n+"
 	@oc project $(NAMESPACE)
-	@oc delete all -l template=$(APP_NAME)-client
+	@oc delete all -l template=$(APP_NAME)-client --ignore-not-found
```

`--ignore-not-found` not needed when filtering by label
```suggestion
	@oc delete all -l app=$(APP_NAME)
```